README.md

```diff
@@ -154,6 +154,8 @@ The `wait_for_selector` option can also be used to wait until an element appears
 
 The `wait_for_timeout` option can also be used to wait the specified number of milliseconds have elapsed.
 
+The `request_failure` option can be used to fail generation if assets fail to load.
```

Comment:
Might be worth expanding here in terms of what is expected when enabled. Something maybe more along the lines of:
```suggestion
The `request_failure` option, when enabled, will raise a `Grover::JavaScript::RequestFailedError`
if the initial content request or any subsequent asset request returns a bad response or times out.
```